### ISSUE TITLE
Can ManipGen Be Adapted for UR Robots Instead of Franka Panda?

### ISSUE THREAD

**Issue by Cheng909865905** [mihdalal/manipgen#12]
We noticed that your implementation uses the Franka Panda robot arm. Would it be feasible to achieve comparable results using a UR series robot instead? If so, are there any specific considerations or modifications required for transferring the policies to UR hardware?

**Comment by minliu01** [comment]
We did not test our pre-trained policies on other embodiments, but I think it would be interesting to see whether it can work with different robots. You might need to match the wrist camera observation and tune the controller carefully.

If zero-shot policy transfer does not work, I think you can reuse our environment, reward design, and training pipeline to obtain policies for UR series robot. If you don't have enough computation resources to train from scratch, you can train RL policies on a subset of our dataset for your robot, and then use DAgger to finetune our pre-trained policies. 

Please feel free to contact me if you would like to try the ideas above or encounter any other issue.
